Dockerfile

```diff
@@ -263,6 +243,33 @@ COPY setup/markdownlint/problem-matcher.json /etc/laminas-ci/problem-matcher/mar
 COPY setup/phpunit/problem-matcher.json /etc/laminas-ci/problem-matcher/phpunit.json
 
 
+# Setup external tools
+COPY composer.json \
+    composer.lock \
+    /tools/
+
+RUN cd /tools \
+    # Install `ext-bcmath` as it seems to be a requirement for `roave/backward-compatibility-check`
+    && apt install -y php-bcmath \
+    && composer install \
```

Comment:
by running this in the final container, we can ensure that all dependencies such as PHP and extension requirements are available since `composer install` does actually not pass in case something is missing.